fe/fe-core/src/main/java/com/starrocks/metric/MetricRepo.java

```diff
@@ -526,6 +542,50 @@ private static void collectTableMetrics(MetricVisitor visitor, boolean minifyTab
         }
     }
 
+    private static void collectRoutineLoadProcessMetrics(MetricVisitor visitor) {
+        List<RoutineLoadJob> jobs = Catalog.getCurrentCatalog().getRoutineLoadManager().getRoutineLoadJobByState(
+                Sets.newHashSet(RoutineLoadJob.JobState.NEED_SCHEDULE, RoutineLoadJob.JobState.RUNNING));
+        for (RoutineLoadJob job : jobs) {
+            if (!(job instanceof KafkaRoutineLoadJob)) {
+                continue;
```

Comment:
Could only collect top 5/10 lag routine loads?

Reply:
We should discuss with @jaogoy